src/utils/devtools-guard.ts

```
@@ -0,0 +1,81 @@
+      e.preventDefault();
+      return;
+    }
+  });
+
+  // Detect DevTools via debugger timing (anti-debug)
+  const detectDevTools = (): void => {
+    const threshold = 160;
+    const start = performance.now();
+    // eslint-disable-next-line no-debugger
+    debugger;
+    const duration = performance.now() - start;
```

Biome (2.3.14)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)